packages/Charts/src/package.ts

```diff
@@ -12,123 +13,158 @@ import {GroupAnalysisViewer} from './viewers/group-analysis/group-analysis-viewe
-  view.addViewer('RadarViewer');
+//name: GlobeViewer
+//description: Creates a globe viewer
+//tags: viewer
+//output: viewer result
+export function _GlobeViewer() {
+  return new GlobeViewer();
 }
 
-//name: TreeViewer
+//name: GroupAnalysisViewer
+//description: Creates a group analysis viewer
 //tags: viewer
-//meta.trellisable: true
 //output: viewer result
-export function _TreeViewer() {
-  return new TreeViewer();
+export function _GroupAnalysisViewer() {
+  return new GroupAnalysisViewer();
 }
 
-//name: SunburstViewer
+//name: RadarViewer
+//description: Creates a radar viewer
 //tags: viewer
 //output: viewer result
-export function _SunburstViewer() {
-  return new SunburstViewer();
+export function _RadarViewer() {
+  return new RadarViewer();
 }
 
 //name: SankeyViewer
+//description: Creates a sankey viewer
 //tags: viewer
 //output: viewer result
 export function _SankeyViewer() {
   return new SankeyViewer();
 }
 
-//name: ChordViewer
+//name: SunburstViewer
+//description: Creates a sunburst viewer
 //tags: viewer
 //output: viewer result
-export function _ChordViewer() {
-  return new ChordViewer();
+export function _SunburstViewer() {
+  return new SunburstViewer();
 }
 
-//name: WordCloudViewer
+//name: SurfacePlot
+//description: Creates a surface plot viewer
 //tags: viewer
 //output: viewer result
-export function _WordCloudViewer() {
-  return new WordCloudViewer();
+export function _SurfacePlot() {
+  return new SurfacePlot();
 }
 
 //name: TimelinesViewer
+//description: Creates a timelines viewer
 //tags: viewer
 //output: viewer result
 export function _TimelinesViewer() {
   return new TimelinesViewer();
 }
 
-//name: SurfacePlot
+//name: TreeViewer
+//description: Creates a tree viewer
 //tags: viewer
+//meta.trellisable: true
 //output: viewer result
-export function _SurfacePlot() {
-  return new SurfacePlot();
+export function _TreeViewer() {
+  return new TreeViewer();
 }
 
-//name: GroupAnalysisViewer
+//name: WordCloudViewer
+//description: Creates a word cloud viewer
 //tags: viewer
 //output: viewer result
-export function _GroupAnalysisViewer() {
-  return new GroupAnalysisViewer();
+export function _WordCloudViewer() {
+  return new WordCloudViewer();
+}
+
+
+//name: chordViewerDemo
+//meta.demoPath: Viewers | Chord
+export function _chordViewerDemo() {
+  Demo.chordViewerDemo();
+}
+
+//name: globeViewerDemo
+//meta.demoPath: Viewers | Globe
+export function _globeViewerDemo() {
+  Demo.globeViewerDemo();
+}
+
+//name: groupAnalysisViewerDemo
+//meta.demoPath: Viewers | GroupAnalysis
+export function _groupAnalysisViewerDemo() {
+  Demo.groupAnalysisViewerDemo();
 }
 
 //name: radarViewerDemo
 //meta.demoPath: Viewers | Radar
 export function _radarViewerDemo() {
-  radarViewerDemo();
+  Demo.radarViewerDemo();
 }
 
-//name: radarViewerDemo
+//name: sankeyViewerDemo
+//meta.demoPath: Viewers | Sankey
+export function _sankeyViewerDemo() {
+  Demo.sankeyViewerDemo();
```

Comment:
Since all the demos are so similar at the moment, would it make sense to simply make one parameterized function viewerDemo(table, options) and simply use it directly here?

Reply:
it makes sense to make one parameterized function, yes
but I guess it's better to use a function like `viewerDemo(viewerName, options)` to transfer the viewer name and options for it
I don't really think that we need the possibility to choose a table in the viewer demo function as the program will choose the appropriate table for the chart by itself

Reply:
guess it's done